lib/logic/controller.js

```diff
@@ -228,6 +228,26 @@ Controller.prototype.open = function(pr) {
     .then(() => this.refresh());
 };
 
+Controller.prototype.waitUntilMergable = function(pr) {
+    var mergeable = false;
+    return this._while(
+        () => !mergeable,
+        () => this.httpApi.getPullRequest(pr.getRepo(), pr.getId())
+                .then((res) => { mergeable = res.body.mergeable; }),
+        200
+    );
+};
+
+Controller.prototype.waitUntilClosed = function(pr) {
+    var closed = false;
+    return this._while(
+        () => !closed,
+        () => this.httpApi.getPullRequest(pr.getRepo(), pr.getId())
+                .then((res) => { closed = res.body.state === "closed"; }),
+        200
```

Comment:
Bumped both; closed seems to actually happen pretty quickly, but mergeable is pretty slow